packages/generator/src/complex-type/field-type-class.ts

```diff
@@ -85,7 +85,7 @@ function property(
 
 export function createPropertyFieldInitializer(prop: VdmProperty): string {
   if (prop.isCollection) {
-    if (prop.isComplex) {
+    if (prop.isComplex || prop.isEnum) {
```

Comment:
👍 I hope we have some kind of test for this? (I think I remember we had something)

Reply:
It's about a collection of enum type in a complex type, which was missing from the test spec file.
I added this case and the generation went well.